# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/pesign/hotfix-install-pm.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- pesign-113/Makefile.vanilla	2022-01-18 10:49:15.290000000 +0100
+++ pesign-113/Makefile	2022-01-18 10:48:43.212000000 +0100
@@ -11,7 +11,7 @@
 
 install :
 	$(INSTALL) -d -m 755 $(INSTALLROOT)$(docdir)/pesign-$(VERSION)/
-	$(INSTALL) -pm 644 COPYING $(INSTALLROOT)$(docdir)/pesign-$(VERSION)/
+	$(INSTALL) -p -m 644 COPYING $(INSTALLROOT)$(docdir)/pesign-$(VERSION)/
 	@set -e ; for x in $(SUBDIRS) ; do \
 		$(MAKE) -C $$x $@ ; \
 	done
